Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -508,15 +522,25 @@ internal async void ClearAllPoolsAsync()
             }
         }
 
-        public void SetMaxPoolSize(int size)
+        public void SetMaxPoolSize(int newMaxPoolSize)
         {
-            _poolConfig.MaxPoolSize = size;
+            s_logger.Info($"SessionPool::SetMaxPoolSize({newMaxPoolSize})");
+            _poolConfig.MaxPoolSize = newMaxPoolSize;
             _configOverriden = true;
         }
 
-        public int GetMaxPoolSize()
+        public int GetMaxPoolSize() => _poolConfig.MaxPoolSize;
+
+        public int GetMinPoolSize() => _poolConfig.MinPoolSize;
```

Comment:
for old pool we don't use minPoolSize. It is returned default value but actually it doesn't matter.
I'm thinking if throwing an exception for parameters not used by given version of pool could be maybe less confusing. But maybe not... I'm just thinking.